strawberry/type.py

```diff
@@ -38,8 +38,10 @@ def _resolver(root, info):
     return _resolver
 
 
-def _process_type(cls, *, is_input=False, is_interface=False, description=None):
-    name = cls.__name__
+def _process_type(
+    cls, *, name=None, is_input=False, is_interface=False, description=None
+):
+    name = name or cls.__name__
```

Comment:
never noticed that we didn't have a way to rename a type. This should be done in another PR :)